docs/basics/index-lifecycle.md

```diff
@@ -0,0 +1,39 @@
+
+Index Lifecycle
+======================
+
+JanusGraph uses only indexes which have status `ENABLED`. 
+When the index is created it will not be used by JanusGraph until it is enabled. 
+After the index is build you should wait until it is registered (i.e. available) by JanusGraph:
+```java
+//Wait for the index to become available (i.e. wait for status REGISTERED)
+ManagementSystem.awaitGraphIndexStatus(graph, "nameAndAgeIndex").call();
+```
+
+After the index is registered we should either enable the index 
```

Comment:
Wouldn't it also be good to show how the index can be enabled without reindexing the data?